lib/tasks_visitor.rb

```diff
@@ -15,17 +15,19 @@ def noop(n, p)
     n
   end
 
-  def visit_hash(n, path)
+  def visit_Hash(n, path)
     n.each do |k, v|
       n[k] = visit(v, path + [k], k)
     end
   end
+  
+  alias :visit_ActiveSupport_HashWithIndifferentAccess :visit_Hash
```

Comment:
Is this alias list exhaustive of the expected types? If not what about using `method_missing` to dispatch the relevant class method?